### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by DenisBogusevich**

### CODE REVIEW

**Review by evgenkolner — APPROVED** [review]
good job

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url" : "jdbc:mysql://mysqldb:$MYSQLDB_DOCKER_PORT/$MYSQLDB_DATABASE?createDatabaseIfNotExist=true",
+        "spring.datasource.username" : "$MYSQLDB_USER",
+        "spring.datasource.password" : "$MYSQLDB_ROOT_PASSWORD",
+        "spring.jpa.hibernate.ddl-auto" : "validate"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (liliia-ponomarenko) [inline docker-compose.yml:30]: add one empty line

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+spring.datasource.password=${MYSQLDB_ROOT_PASSWORD}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 
-jwt.expiration=5000000
-jwt.secret=secretgfhjghjhghjgjgfccjgvcgfxhdxxhfxszzgxvjhvhv
+jwt.expiration=${JWT_EXPIRATION}
+jwt.secret=${JWT_SECRET}
+
```
Comment (liliia-ponomarenko) [inline src/main/resources/application.properties:14]:
```suggestion
```

File: .gitignore [hunk 3]
```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+### Docker ###
+.env
```
Comment (liliia-ponomarenko) [inline .gitignore:36]:
```suggestion
.env

```

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! But fix a few mistakes before merging ;)

**Pull request merged**